File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.py
class Solution:
    def countConsistentStrings(self, allowed: str, words: List[str]) -> int:
        cnt = 0

        for word in words:
            flag = 1
            for ch in word:
                if ch not in allowed:
                    flag = 0
                    break
            if flag:
                cnt += 1
                

        return cnt

File: 1786-count-the-number-of-consistent-strings/count-the-number-of-consistent-strings.cpp
class Solution {
public:
    int countConsistentStrings(string allowed, vector<string>& words) {
        int arr[26] = {0};
        for(char ch: allowed) {
            arr[ch - 'a'] = 1;
        }

        int cnt = 0;

        for (string& word: words) {
            int flag = 1;
            
            for (char ch: word) {
                if (arr[ch - 'a'] == 0) {
                    flag = 0;
                    break;
                }
            }

            cnt += flag;
        }

        return cnt;
    }
};